stable/jenkins/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: jenkins
 home: https://jenkins.io/
-version: 0.16.3
+version: 1.0.0
```

Comment:
I wouldn't make it a 1.0.0. The chart would have to be updated to follow our best practices first.

@viglesiasce WDYT?

Reply:
@unguiculus I updated to 1.0.0 because there is a breaking change.

The semver definition:

```
Semantic Versioning 2.0.0
Summary
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards-compatible manner, and
PATCH version when you make backwards-compatible bug fixes.
```